lib/qesapdeployment.pm

```diff
@@ -490,7 +490,23 @@ sub qesap_execute {
 
     my $exec_rc = qesap_venv_cmd_exec(cmd => $qesap_cmd, timeout => $args{timeout}, failok => 1);
 
+    my @qesap_logs;
+    my $qesap_log_find = 'find . -type f -name "*.log.txt"';
```

Comment:
The thing I don't like about this is that if for some reason glue script fails to generate logs, then this will not find any file and silently fail. Can't we generate the list of logs from somewhere else?

Reply:
yes, it is intentional:

- I like dependency between test code and internal glue script implementation to be minimal. At the moment is it as simple as "Whatever is generated in current dir and with extension .log.txt". I could eventually keep it more restrictive using `"$qesap_cmd.*\.log\.txt"` (here I'm proposing a sort of regexp using value from a perl variable, but probably the result  is not something valid for the find command. Just keep it as an idea)
- I do not want the test to fails if no script are generated, for whatever glue script internal reason/change/bug